tracer/test/Datadog.Trace.Security.Unit.Tests/IAST/Tainted/VulnerabilitiesEnumTests.cs

```diff
@@ -42,7 +42,7 @@ private static List<object[]> GetTestData()
         var jsonContent = ResourceHelper.ReadAllText<VulnerabilitiesEnumTests>("vulnerability_schema.json");
         var jsonSchema = JObject.Parse(jsonContent);
 
-        var vulnerabilityTypes = (JArray)jsonSchema.SelectToken("properties.vulnerabilities.items.properties.type.enum");
+        var vulnerabilityTypes = (JArray)jsonSchema.SelectToken("definitions.VulnerabilityType.enum");
```

Comment:
Minor: don't see how this change is related.

Reply:
The file vulnerability_schema.json was updated with it's latest version and there was a format change in it that requires this change in the unit tests.

https://github.com/DataDog/dd-trace-dotnet/pull/5780/#discussion_r1673695194